provider/source/src/time_zones/mod.rs

```diff
@@ -272,28 +274,63 @@ impl SourceDataProvider {
                     .map(|(&region, iana)| (iana, region))
                     .collect::<BTreeMap<_, _>>();
 
-                let bcp47_tzids = self.bcp47_to_canonical_iana_map()?;
-
-                let zone_tab = self.tzdb()?.zone_tab()?;
-                let mut zones_per_region: BTreeMap<icu::locale::subtags::Region, usize> =
-                    BTreeMap::new();
+                let bcp47_tzids_resource = &self
+                    .cldr()?
+                    .bcp47()
+                    .read_and_parse::<cldr_serde::time_zones::bcp47_tzid::Resource>(
+                        "timezone.json",
+                    )?
+                    .keyword
+                    .u
+                    .time_zones
+                    .values;
 
-                for &region in bcp47_tzids.values().flat_map(|iana| zone_tab.get(iana)) {
-                    *zones_per_region.entry(region).or_default() += 1;
+                let mut regions_to_zones: BTreeMap<_, BTreeSet<_>> = BTreeMap::new();
+
+                for (&bcp47_tzid, bcp47_tzid_data) in bcp47_tzids_resource {
+                    regions_to_zones
+                        .entry(match bcp47_tzid.0.as_bytes() {
+                            // backfill since this data is not in 47 yet
+                            b"ancur" => region!("CW"),
+                            b"fimhq" => region!("AX"),
+                            b"gpmsb" => region!("MF"),
+                            b"gpsbh" => region!("BL"),
+                            b"gazastrp" | b"hebron" => region!("PS"),
+                            b"jeruslm" => region!("IL"),
```

Comment:
But it is new in 47? How is this data missing?

Reply:
it's 48